tox.ini

```diff
@@ -7,7 +7,7 @@ setenv = COVERAGE_FILE={toxworkdir}/.coverage.{envname}
 usedevelop = True
 deps = -rrequirements.txt
        -rtests/requirements.txt
-commands = pytest {posargs:-vv --cov=src/ --cov-report=xml tests/}
+commands = pytest {posargs:-vv --cov=src/ --cov=tests/ --cov-report=xml tests/}
```

Comment:
The addition of `--cov=tests/` allows to spot when parts of `tests/` are never executed anymore (and can be cleaned up)